api/lib/Gateway.ts

```diff
@@ -2370,6 +2370,36 @@ export default class Gateway {
 		}
 	}
 
+	private _availabilityConfig(
```

Comment:
Only a naming preference: 
```suggestion
	private setDiscoveryAvailability(
```

Reply:
Done. Please keep me on my toes style-wise... I'm going to have a few more PRs, and TypeScript (and JavaScript) is definitely not my first language.